api/update.go

```diff
@@ -105,6 +107,7 @@ func (api API) Update(w http.ResponseWriter, r *http.Request) {
 			Message:           "Domain was added and removed without being preloaded.",
 			Status:            database.StatusRejected,
 			IncludeSubDomains: entry.IncludeSubDomains,
+			Policy:            entry.Policy,
```

Comment:
I'm noticing that a lot of these look very similar (copy a bunch of fields from the `preloadlist.Entry` and set a Status). Optionally, it might be nice to have an equivalent to `DomainState.ToEntry` that goes the other way and takes in a status.

Reply:
Done.